src/components/Home.jsx

```diff
@@ -0,0 +1,173 @@
+import React from "react";
+import NavBar from "./NavBar";
+import { RiDeleteBin6Line } from "react-icons/ri";
+import {
+  MdOutlineColorLens,
+  MdOutlineArchive,
+  MdOutlineLabel,
+} from "react-icons/md";
+function Home() {
+  return (
+    <div>
+      <NavBar />
+      <div class="home-cont">
+        <aside class="aside-cont">
+          <ul class="list list-stack">
+            <li class="list-item">
+              <svg
+                xmlns="http://www.w3.org/2000/svg"
```

Comment:
change class to className in all the places in this file.

Reply:
sure manis thanks